AntViewer/Classes/Models/Poll.swift

```diff
@@ -83,7 +84,11 @@ public class Poll {
       }
       
       if let id = User.current?.id {
-        self?.answeredByUser = documents.contains(where: {$0.documentID == "\(id)"})
+        let userDocument = documents.first(where: { $0.documentID == "\(id)" })
+        self?.answeredByUser = userDocument != nil//documents.contains(where: {$0.documentID == "\(id)"})
```

Comment:
do we need this property now?

Reply:
sure
It's related to behaviour of pollBanner(when user has already made choose, we don't have to show expanded pollBanner)

Reply:
Done.